pkg/connectors/datacatalog/clients/datacatalog_openapi.go

```diff
@@ -39,17 +40,18 @@ func NewOpenAPIDataCatalog(name string, connectionURL string, connectionTimeout
 	return &openAPIDataCatalog{
 		name:   name,
 		client: apiClient,
+		log:    log,
 	}
 }
 
 func (m *openAPIDataCatalog) GetAssetInfo(in *datacatalog.GetAssetRequest, creds string) (*datacatalog.GetAssetResponse, error) {
 	resp, r, err := m.client.DefaultApi.GetAssetInfoPost(context.Background()).XRequestDataCatalogCred(creds).DataCatalogRequest(*in).Execute()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error when calling `DefaultApi.GetAssetInfoPost``: %v\n", err)
-		fmt.Fprintf(os.Stderr, "Full HTTP response: %v\n", r)
+		m.log.Error().Err(err).Msg("error when calling `DefaultApi.GetAssetInfoPost`")
```

Comment:
Please add the datasetid and uuid

Reply:
Added datasetID. It is a problem to add uuid without modifying the open-api data catalog interface.

Reply:
Should we open an issue for this, or take care of it in this PR?

Reply:
I don't think the printout should be here. It should be in the controller upon receiving the error. I'll change the code accordingly.

Reply:
Changed the code to include uuid and datasetID in the logs